reference/ft_megplanar.md

```diff
@@ -4,13 +4,13 @@ title: ft_megplanar
 ```plaintext
  FT_MEGPLANAR computes planar MEG gradients gradients for raw data or average
  event-related field data. It can also convert frequency-domain data that was computed
- using FT_FREQANALYSIS, as long as it contains the complex-valued fourierspcrm and not
+ using FT_FREQANALYSIS, as long as it contains the complex-valued fourier and not
```

Comment:
I would say that this here should read 'fourierspctrm', because that is the proper name of the field that contains the fourier coefficients if cfg.output was 'fourier'..., could you please update?